packaging/docker/build-enterprise.sh

```diff
@@ -10,16 +26,33 @@ else
   _grafana_tag="${_raw_grafana_tag}"
 fi
 
-echo "Building and deploying ${_docker_repo}:${_grafana_tag}"
+if [ ${UBUNTU_BASE} = "0" ]; then
```

Comment:
I'd make this section into a function instead of duplicating the code. Unless there's something that's different between the two sections that I'm missing.

Reply:
@xlson I got rid of the code duplication now by using a couple of variables, which worked better for me than making a function. Please have another look.

Reply:
That looks a lot better. Thanks for cleaning that up.